index.html

```diff
@@ -410,10 +458,10 @@
       execution behaviors without incurring the penalty of delayed resource
       loading.</p>
       <p>For example, <a>preload link</a> enables the application to provide
-      `async` and `defer` like semantics, which are only available on `script`
+      <code>async</code> and <code>defer</code> like semantics, which are only available on <code>script</code>
```

Comment:
Same here and for the rest of. You should never need `<code>` - please convert these to `<a>`.